content/200-concepts/100-components/02-prisma-client/056-composite-types.mdx

```diff
@@ -177,6 +178,18 @@ const orders = await prisma.order.findMany({
 })
 ```
 
+Use `isSet` to filter for orders where the optional `billingAddress` has been set (either to a value or to `null`):
```

Comment:
"either to a value or to `null`" => "this also includes fields that are explicitly set to `null`"

Reply:
Done